src/main/webapp/WEB-INF/view/profilepage.jsp

```diff
@@ -18,8 +18,11 @@
             <img id="profilePicture" src="https://i.imgur.com/z4amwTY.png">
         </div>
         <div id="leftContainer">
-            <h1>About Me</h1>
-            <textarea id="aboutMe" rows='25'></textarea>
+            <form action="/profilepage">
```

Comment:
I know it's just a little UI change, but can you add a screenshot of the new submit button?

You could also consider keeping the UI change separate from the new setter methods since, even though they're both small, they're not directly related to each other. I'm also kind of curious what happens when you press the button in the UI (is it just nothing?), since there isn't a profilepage post method yet.

Reply:
<img width="1248" alt="screen shot 2018-04-12 at 12 01 25 am" src="https://user-images.githubusercontent.com/25258786/38655830-cba44552-3de4-11e8-88ba-74e4dbf278f5.png">

Yeah, the button isn't too fancy yet :) But when you press it, it just "refreshes" the form and submits it to nowhere. I was thinking that we could grab the "aboutMe" variable in the user profile and display it in the text box and when the button is pressed it will update this display then?

Reply:
Thats how it should work is what i think, I just realized i forgot to add a box for interests (although updating interests would be with a special menu because of the map structure)

Reply:
What do you think about having a drop down input for the category (and we can hardcode different categories) then they can type in a more specific interest?

Reply:
I think thats fine, although that sounds like a menu to add new interests instead of the actual interests display

Reply:
Yeah, we could have sort of a menu bar style to suggest categories. It would definitely seem more organized but it depends how much freedom we want them to have with categories I guess.